### PR TITLE
Speed up bulk upload

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

File: seqr/views/apis/phenotips_api.py [hunk 1]
```diff
@@ -62,29 +62,20 @@ def create_patient(project, individual):
...
+    headers = {"Content-Type": "application/json"}
+    data = json.dumps({'external_id': individual.guid})
     auth_tuple = _get_phenotips_uname_and_pwd_for_project(project.phenotips_user_id)
-    _make_api_call('GET', url, auth_tuple=auth_tuple, verbose=False, parse_json_resonse=False)
 
-    #data = {}
-    #if patient_json is not None:
-    #    data = json.dumps(patient_json)
+    response_items = _make_api_call('POST', url, auth_tuple=auth_tuple, http_headers=headers, data=data, expected_status_code=201, parse_response_items=True)
+    patient_id = response_items['Location'].split('/')[-1]
```
Comment (bw2) [inline seqr/views/apis/phenotips_api.py]: Nice find with using response headers to get the id.

File: seqr/views/apis/pedigree_image_api.py [hunk 2]
```diff
@@ -105,7 +104,7 @@ def update_pedigree_image(family):
 
         # columns: family, individual id, paternal id, maternal id, sex, affected
         for i in individual_records.values():
-            row = [i[key] for key in ['familyId', 'individualId', 'paternalId', 'maternalId', 'sex', 'affected']]
+            row = [family_id] + [i[key] for key in ['individualId', 'paternalId', 'maternalId', 'sex', 'affected']]
```
[the remark this answers is not in the document]
Comment (hanars) [reply seqr/views/apis/pedigree_image_api.py]: `familyId` is no longer a field on that object though, it added an unnecessary db query and we already have it

File: seqr/views/apis/phenotips_api.py [hunk 3]
```diff
@@ -62,29 +62,20 @@ def create_patient(project, individual):
...
 
-    #auth_tuple = _get_phenotips_uname_and_pwd_for_project(project.phenotips_user_id)
-    #_make_api_call('POST', url, data=data, auth_tuple=auth_tuple, verbose=False, parse_json_resonse=False)
-
-    patient_data = get_patient_data(project, individual)
-    patient_id = patient_data['id']
-
-    username, _ = _get_phenotips_uname_and_pwd_for_project(project.phenotips_user_id)
```
[the remark this answers is not in the document]
Comment (hanars) [reply seqr/views/apis/phenotips_api.py]: when creating via the api endpoint, the authenticated user is automatically added as an owner

File: seqr/views/apis/phenotips_api.py [hunk 4]
```diff
@@ -62,29 +62,20 @@ def create_patient(project, individual):
     Raises:
         PhenotipsException: if unable to create patient record
     """
-    url = '/bin/PhenoTips/OpenPatientRecord?create=true&eid={patient_eid}'.format(patient_eid=individual.guid)
 
+    url = '/rest/patients'
```
[the remark this answers is not in the document]
Comment (hanars) [reply seqr/views/apis/phenotips_api.py]: yes

**Pull request merged**